provider/k8s/template/app/service.yml.tmpl

```diff
@@ -107,6 +107,28 @@ spec:
       serviceAccountName: {{.Service.Name}}
       shareProcessNamespace: {{.Service.Init}}
       terminationGracePeriodSeconds: {{$.Service.Termination.Grace}}
+      {{if .Service.InitContainer.Command }}
```

Comment:
it should be depends on the command, it should be depend on the whether InitContainer defined or not